pkg/asset/machines/vsphere/machines.go

```diff
@@ -63,7 +63,7 @@ func Machines(clusterID string, config *types.InstallConfig, pool *types.Machine
 	return machines, nil
 }
 
-func provider(clusterID string, platform *vsphere.Platform, mpool *vsphere.MachinePool, osImage string, userDataSecret string) (*vsphereapis.VSphereMachineProviderSpec, error) {
+func provider(clusterID string, platform *vsphere.Platform, mpool *vsphere.MachinePool, 	 string, userDataSecret string) (*vsphereapis.VSphereMachineProviderSpec, error) {
```

Comment:
@patrickdillon looks like you started to edit the name...

Reply:
woops. fixed.